packages/slice-machine/test/__setup__.ts

```diff
@@ -20,7 +21,11 @@ declare module "vitest" {
   }
 }
 
-const mswServer = setupServer();
+const mswServer = setupServer(
```

Comment:
MSW prints the following error when running some tests (e.g. `CustomTypeBuilderPage.test.tsx`):

```
[MSW] Error: captured a request without a matching request handler:

  • GET https://api.lab.amplitude.com/sdk/v2/vardata

If you still wish to intercept this unhandled request, please create a request handler for it.
Read more: https://mswjs.io/docs/getting-started/mocks
[Experiment] Retry falied:  Error: [MSW] Cannot bypass a request when using the "error" strategy for the "onUnhandledRequest" option.
    at applyStrategy ([redacted]/slice-machine/node_modules/msw/src/utils/request/onUnhandledRequest.ts:234:1)
    at onUnhandledRequest ([redacted]/slice-machine/node_modules/msw/src/utils/handleRequest.ts:73:3)
    at handleRequest ([redacted]/slice-machine/node_modules/msw/src/utils/handleRequest.ts:127:3)
    at [redacted]/slice-machine/node_modules/msw/src/node/SetupServerApi.ts:107:9
```

This seems to happen because `@amplitude/experiment-node-server` is not mocked in `slice-machine-ui` tests. My Vitest breadcrumb tests were failing while I was writing this PR, and mocking the endpoint seemed to fix it.

However, it's working now without the extra network mock, so I guess we should just leave it as it was. My code may have been in a weird state at the time.

Reply:
Besides, we are planning to remove most of the Vitest tests anyway, so I don't think we need to worry too much about these errors/warnings.